.gitignore

```diff
@@ -5,7 +5,7 @@ pkg/
 spec/dummy/db/*.sqlite3
 spec/dummy/db/*.sqlite3-journal
 spec/dummy/log/*.log
-spec/dummy/tmp/
```

Comment:
you want the `spec/dummy/tmp/` (in fact, it'd be appropriate to prefix all these spec lines with a `/` - you then want to explicitly allow .keep with a `.keep` line.